src/Module.Server/Rewards/CrpgRewardServer.cs

```diff
@@ -243,7 +243,7 @@ public override void OnRemoveBehavior()
             if (missionPeer.Team != null && missionPeer.Team.Side != BattleSideEnum.None)
             {
                 int crpgUserId = crpgPeer.User.Id;
-                var crpgUserDamagedItems = BreakItems(crpgPeer, durationRewarded * (isLowPopulationServer ? 0.2f : 1f));
+                var crpgUserDamagedItems = BreakItems(crpgPeer, durationRewarded * (isLowPopulationServer ? 0f : 1f));

```

Comment:
You can avoid the call to GetBrokenItemsByCrpgUserId altogether when it's low pop

Reply:
it's done